lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -176,8 +188,130 @@ export default class BitcoinProcessor {
   }
 
   /**
-   * Trims the databases to the block prior to the block the given transaction is in.
-   * @returns The last fully processed block after trimming. `undefined` if all data are deleted after trimming.
+   * A faster version of process transactions that requires access to bitcoin data directory
```

Comment:
Sanity: I am a bit concerned that in the event that the node is terminated unexpectedly halfway during this bootstrapping method, when it comes back up it will incorrectly think that the node is sync to a certain transaction (based on the last transaction in DB), and incorrect pick up from there resulting in bad state.

Reply:
Created an issue for it https://github.com/decentralized-identity/sidetree/issues/794